control-plane/aegisops_control_plane/config.py

```
@@ -1,14 +1,84 @@
+            for component in normalized_path.split("/")
+            if component
+        )
+        secret_url = f"{normalized_address}/v1/{encoded_mount}/data/{encoded_path}"
+        backend_request = request.Request(
+            secret_url,
+            headers={
+                "Accept": "application/json",
+                "X-Vault-Token": token,
+            },
+            method="GET",
+        )
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)